src/arena/cfr/action_generator.rs

```diff
@@ -24,7 +24,7 @@ pub trait ActionGenerator {
     ///
     /// The index of the action in the children array. The 0 index is the fold
     /// action. All other are defined by the implentation
-    fn action_to_idx(&self, game_state: &GameState, action: &AgentAction) -> usize;
+    fn action_to_idx(&self, action: &AgentAction) -> usize;
```

Comment:
We really do need the `ActionGenerator` passed in. 

The simple version of poker that we have where the player can only fold, call, or go all in is just a simplification. A more accurate implementation (now that we know the issue) would be different for pre-flop when there can be more players, vs later in the game where there are fewer players.